### PR TITLE
Courses #1 (Kian, Cheyenne)

### PR DESCRIPTION
**Pull request by KianBaghai**
## Description
Made a new courses page and added a link to the first course

### CODE REVIEW

**Review by joycechen721 — CHANGES_REQUESTED** [review]

File: src/app/courses/opiod/page.tsx [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (coshio) [reply src/app/courses/opiod/page.tsx]: this was resolved in following commits :)

**Comment by WCheyenneLu** [comment]
# Courses-1 (Kian, Cheyenne)
## Description:
- On main home page for courses, each course is displayed using the Course component
- Each course has two modules (for now) which has multiple pages (dynamic routing)
- Pages have two buttons that help navigate to previous page and next page (useParams, url coerced into number to add and subtract 1) 
- timer created for a simple page so that users have to have the page open for at least 10 seconds before moving on to the next
## Components:
### Course.tsx
  **Functionality**: 
- takes in coursePath, courseProgress, courseTitle props from the course home page, onClick navigates to the respective course's page
- displays progress as progress bar with <progess> tag
### Module.tsx
  **Functionality**: 
- takes in coursePath, courseProgress, courseTitle props from the course (career or opioid) page, onClick navigates to the first page of the course
- displays progress as progress bar with <progess> tag
### SimplePage.tsx
  **Functionality**: The basic structure of a 'standard' page, requires user to stay on that page for at least 10 seconds before they are able to move one (functional timer state that stops at 10 but further implementations will be executed in the future)
### VideoPage.tsx
  **Functionality**: The basic structure of a page in the module that may have video content
## To Do:
- Add percentage to progress bar
- Add props to simple page so correct information displayed, use the timer state for rendering next page options

**Comment by cheyennelu17** [comment]
# Courses-1 (Kian, Cheyenne)
## Description:
- On main home page for courses, each course is displayed using the Course component
- Each course has two modules (for now) which has multiple pages (dynamic routing)
- Pages have two buttons that help navigate to previous page and next page (useParams, url coerced into number to add and subtract 1) 
- timer created for a simple page so that users have to have the page open for at least 10 seconds before moving on to the next
## Components:
### Course.tsx
  **Functionality**: 
- takes in coursePath, courseProgress, courseTitle props from the course home page, onClick navigates to the respective course's page
- displays progress as progress bar with <progess> tag
### Module.tsx
  **Functionality**: 
- takes in coursePath, courseProgress, courseTitle props from the course (career or opioid) page, onClick navigates to the first page of the course
- displays progress as progress bar with <progess> tag
### SimplePage.tsx
  **Functionality**: The basic structure of a 'standard' page, requires user to stay on that page for at least 10 seconds before they are able to move one (functional timer state that stops at 10 but further implementations will be executed in the future)
### VideoPage.tsx
  **Functionality**: The basic structure of a page in the module that may have video content
## To Do:
- Add percentage to progress bar
- Add props to simple page so correct information displayed, use the timer state for rendering next page options

**Review by coshio — APPROVED** [review]
Great work you guys and really good PR documentation

**Review by coshio — APPROVED** [review]

**Review by joycechen721 — APPROVED** [review]
looks gassss THANKS GUYS

**Pull request merged**